### DISCUSSION TITLE
What is the unit of Berry curvature?

### DISCUSSION THREAD

**Discussion by AbhirupRK** [wannier-berri/wannier-berri#180] [Q&A] [answered]
k-points are taken in reduced coordinates. So what is the unit of Berry curvature? Is it dimensionless ?

Also, if I integrate BC  to get AHC, then what would be the unit of AHC ?

**Comment by stepan-tsirkin** [comment, marked as answer]
Berry curvature is evaluated in cartesian coordinates, so it has units length squared , which in WannierBerri is  Angstroms$^{2}$

The AHC has conductivity onits, which is in present version Siemens/m , but until recently it was Siemens/cm, so be careful with that.

**Reply by Bhuvaneswari-R432** [reply]
Dear sir,

In 2D, what will be the unit of Berry curvature dipole? Is it Angstrom? (since BCD behaves as a pseudo-vector in 2D plane)

**Reply by stepan-tsirkin** [reply]
In principle it is. Nut the code in fact does not calculate 2D, it calculates the slab, so the result is still dimensionless. To get the 2D quantity, just multiply by the period of the slab geometry.
